introduction.md

```diff
@@ -0,0 +1 @@
+You just entered the dojo, here is a repo for some rules to know. Welcome.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.